data/en/cta.yml

```diff
@@ -3,9 +3,9 @@ cta:
   enable : true
   bg_image : "images/backgrounds/clouds.jpg"
   bg_image_webp : "images/backgrounds/clouds.webp"
-  title : "Drawing Strength from Scale"
-  content : "Supporting geo-distributed companies, creators, and communities in the modern digital landscape."
+  title : "Drawing Flexibility | Resiliency | Security"
```

Comment:
@KNGJOEL Please remove the word "Drawing". It should state: "Flexibility | Resiliency | Security"

Reply:
Got it

Reply:
@KNGJOEL @eschmier looks like this is only change left to be made?